.pre-commit-config.yaml

```diff
@@ -16,4 +16,4 @@ repos:
   rev: ''  # Use the sha / tag you want to point at
   hooks:
     - id: mypy
-      additional_dependencies: [numpy, typing_extensions]
\ No newline at end of file
+      additional_dependencies: [numpy, typing_extensions]
```

Comment:
These changes will be reverted once everything else looks good. I added them so that pre-commit would run for me.